### ISSUE TITLE
Field mismatch with to response.

### ISSUE THREAD

**Issue by abubakar364** [slackapi/java-slack-sdk#1267]
In slack user model we have variable
  private String teamId;
  
However the API response from slack is team_id. So when we call the API in JAVA is says no variable found against team_id; How can I handle that ?

**Comment by seratch** [comment]
Hi @abubakar364, thanks for asking the question. This SDK maps the snake_cased field names to camelCased names to follow the common standard in Java programming. So, all you need to do is just to use getTeamId() of an object.

**Comment by abubakar364** [comment]
Hi @seratch  Thank you so much for the clarification. One more question is there a way we can fetch the channel Id using receiver Id ?

**Comment by seratch** [comment]
What do you mean by the "receiver" id? If that's a user ID in an event payload and you'd like to know the channel ID for the DM with the person, you can perform conversations.open API call to create a new one or get the ID if it already exists. If you meant a different scenario, could you elaborate on that more?

**Comment by abubakar364** [comment]
Yeah, So what my use case is I'm designing a chat app using Slack. I have shown the list of users. I can send messages using userID in the payload. However, when I have to fetch the history of that channel I don't have the channel ID. So is there a way I can get a channel ID for the DM ? or any better approach for my use case.

**Comment by seratch** [comment]
Okay, so as I mentioned above, you can call conversations.open API to know the DM channel ID. For 1:1 DM with your app's bot user, you can pass a single user ID as the "users" parameter: https://api.slack.com/methods/conversations.open

If everything is clear now, would you mind closing this issue?

**Comment by abubakar364** [comment]
Yup sure Thanks
